src/lib/Output/Generator/Json/FieldTypeHashGenerator.php

```diff
@@ -126,6 +141,33 @@ protected function isNumericArray(array $value)
 
         return true;
     }
+
+    /**
+     * @param \Ibexa\Rest\Output\Generator\Json\ArrayObject|\Ibexa\Rest\Output\Generator\Json\JsonObject $parent
+     *
+     * @return mixed
+     */
+    private function generateObjectValue($parent, object $value)
```

Comment:
`$parent` here could have strict type instead PHPDoc, right?

Reply:
No, these two classes `JsonObject` and `ArrayObject` are unrelated. Since we cannot use class unions (`JsonObject|ArrayObject`) we cannot strictly type here.